editor/editor_node.cpp

```diff
@@ -5536,6 +5537,10 @@ EditorNode::EditorNode() {
 		import_texture_atlas.instance();
 		ResourceFormatImporter::get_singleton()->add_importer(import_texture_atlas);
 
+		Ref<ResourceImporterCSV> import_csv;
```

Comment:
Since its registered first, does it become the default import preset? If so that shouldbe changed, as importing CSV for translations is still the main workflow for now (changing it breaks compat).

Reply:
Yes, I had done it that way on purpose since doing it the other way means that anyone who just copy/pastes a CSV will result in errors showing up in their console. But I see what you mean about it breaking compatibility, so I guess I'll switch it back around. It's not a big deal for users if they have to clear out some errors in the console and re-import their CSV, but it is a big deal if import data on a translation is lost somehow.